__tests__/app.test.js

```diff
@@ -0,0 +1,48 @@
+const { expect } = require("@jest/globals");
+const request = require("supertest");
+const app = require("../app/app");
+const db  = require("../db/connection");
+const dataTest = require("../db/data/test-data/index")
+const seed = require("../db/seeds/seed")
+
+beforeEach(() => {
+    return seed(dataTest);
+})
+
+afterAll(() => {
+    db.end();
+})
+
+describe('GET /api/categories', () => {
+    test('getAPI should return 200 status and a message', () => {
```

Comment:
You can wrap the 200 test into other tests as you have done on line 29 - no need to have a standalone test just for 200 status. This will save you some typing 😄